### PR TITLE
Add get default perspective test

### PR DESCRIPTION
**Pull request by annyhe**
- on GET default perspective: if the perspective exists, redirect to that perspective using the key value in the response.
- add tests to show what happens on GET default perspetive

### CODE REVIEW

**Comment by annyhe** [comment]
DO NOT merge. the problem is window.location.href = '...' does not redirect until the end of the function, so if there is a default perspective, the url will be routed to the FIRST perspective, instead of the default.

**Comment by annyhe** [comment]
This PR shows the default perspective is loaded correctly, if it exists. You can test it out at https://refocus-devbox.herokuapp.com/perspectives/

**Comment by annyhe** [comment]
This is what the /perspectives page looks like, with NO perspectives
screen shot 2017-06-14 at 8 41 49 am

**Comment by annyhe** [comment]
Updated view when no perspectives exist
screen shot 2017-06-14 at 8 47 22 am

File: view/perspective/utils.js [hunk 1]
```diff
@@ -347,49 +344,64 @@ function getValuesObject(accumulatorObject) {
...
+     * GET default perspective: does NOT exist: perspective is the first
+     *  perspective in the perspectives array.
+     * If no perspectives exist, valuesObj.perspective = null
+     */
+     if (!named) {
+      if (returnedPerspective) {
+
+        // the value field has the name of the default perspective.
+        // need the return statement to skip executing the rest of code
+        return redirectToUrl('/perspectives/' + returnedPerspective.value);
```
Comment (pallavi2209) [inline view/perspective/utils.js]: Always a good practice to check for attribute before using it.

File: view/perspective/utils.js [hunk 2]
```diff
@@ -347,49 +344,64 @@ function getValuesObject(accumulatorObject) {
...
+      }
+
+      // default perspective does NOT exist.
+      // GET the first perspective by alphabetical order.
+      // Check to see there are perspectives
+      if (valuesObj.perspectives.length) {
+
+        // redirect to the first perspective. The rest of the code
+        // won't be executed.
+        return redirectToUrl('/perspectives/' + valuesObj.perspectives[0].name);
```
Comment (pallavi2209) [inline view/perspective/utils.js]: Check for valuesObj.perspectives[0].name. Always a good practice to check for attribute before using it.

Comment (annyhe) [inline view/perspective/utils.js]: I put this in another story.

File: view/perspective/utils.js (hunk 1 above)
Comment (annyhe) [inline view/perspective/utils.js]: I put this in another story.

**Pull request merged**